usaspending_api/api_contracts/contracts/award/idv/Funding.md

```diff
@@ -13,12 +13,12 @@ Returns File C financial data for an IDV (Indefinite Delivery Vehicle) award's d
             Either a "generated" natural award id (string) or a database surrogate award id (number).  Generated award identifiers are preferred as they are effectively permanent.  Surrogate award ids are retained for backward compatibility but are deprecated.
         + `piid`: `15B30518FTM230002` (optional, string)
             `Award ID` to further refine results.  All File C financial data for this award is returned if omitted.
-        + `limit`: 5 (optional, number)
+        + `limit`: `5` (optional, number)
```

Comment:
numbers don't need to be quoted

Reply:
As we discussed, numbers don't NEED backticks, but it doesn't hurt anything.  Backticks are used for highlighting in markdown and escaping certain characters in MSON.  I thought it would be nice to consistently highlight values.  Clearly I was mistaken.  They have been removed for numbers.